apps/src/p5lab/spritelab/SpriteLab.js

```diff
@@ -59,3 +60,23 @@ SpriteLab.prototype.onPause = function(isPaused) {
     coreLibrary.startPause(current);
   }
 };
+
+SpriteLab.prototype.setupReduxSubscribers = function(store) {
```

Comment:
question + comment:
1. where/how is this function called?
2. this should probably have unit tests

Reply:
It's called here during P5Lab setup: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/P5Lab.js#L325
In this function, we also call through to the parent function defined here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/P5Lab.js#L611

Do you know of any examples of how to test something like this? I looked at existing applab and gamelab tests and we don't have any coverage of redux stuff.

Reply:
we would probably have to stub all of the redux stuff, which i think is fine -- the test would just make sure that our customEvent is only dispatched when we expect it to be. 

i don't see any examples for stubbing this exact scenario, but since `store` is a param for this function, we could do something like:

```javascript
const store = {
  subscribe: callback => callback()
};
spriteLab.setupReduxSubscribers(store);
```